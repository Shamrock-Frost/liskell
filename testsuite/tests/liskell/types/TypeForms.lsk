(defmodule TypeForms _ ())

(define main undefined)

(define regularTyApp undefined (Either Int Char))

(define funType undefined
  (-> Int Bool Char))

(define contextedType undefined
  (=> ((Show a) (Num a))
      (-> a String)))

(define contextedTypeAutoWrapped undefined
  (=> (Show a)
      (-> a String)))

(define listType undefined
  ([] Int))

(define nullarytupType undefined
  (,))

(define twotup undefined
  (, Int Bool))

(define nonstandardapp undefined
  (app Either Int Bool))

(defwithsig
    (myid a)
    (-> a a)
    a)

;(defdata Foo ()) - Bang Type missing