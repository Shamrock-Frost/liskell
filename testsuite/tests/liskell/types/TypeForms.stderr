[1 of 1] Compiling TypeForms        ( TypeForms.lsk, TypeForms.o )
Warning: output was redirected with -o, but no output will be generated
because there is no Main module.
